src/app/community/_components/PostCard.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { useState } from 'react';
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+
+import ContentImage from '@/public/images/myProfile.jpeg';
+import { CommunityCardDataType } from '@/app/mj/communityData';
+import { calculateTimeDifference } from '@/libs/calculateDate';
+
+import { Modal } from '@/components';
+import AuthorCard from './AuthorCard';
+import { PostInteractions } from './PostInteractions';
+
+import styles from './PostCard.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface PostCardProps {
+  cardData: CommunityCardDataType;
+}
+
+export default function PostCard({ cardData }: PostCardProps) {
+  const [isPostModalOpen, setIsPostModalOpen] = useState(false);
+  const handleClickPostModal = () => {
+    setIsPostModalOpen(true);
+  };
+  const handleClosePostModal = () => {
+    setIsPostModalOpen(false);
+  };
+  const {
+    user_nickname: nickname,
+    created_at: createdAt,
+    title,
+    image,
+    good_count: goodCount,
+    comment_count: commentCount,
+  } = cardData;
+  const nowDate = new Date();
+  const createdDate = new Date(createdAt);
+  const timeToString = calculateTimeDifference(createdDate, nowDate);
+
+  return (
```

Comment:
```suggestion
export default function PostCard({ cardData }: PostCardProps) {
   const [isPostModalOpen, setIsPostModalOpen] = useState(false);
   
   const {
    user_nickname: nickname,
    created_at: createdAt,
    title,
    image,
    good_count: goodCount,
    comment_count: commentCount,
  } = cardData;
  
  const nowDate = new Date();
  const createdDate = new Date(createdAt);
  const timeToString = calculateTimeDifference(createdDate, nowDate);
    
  const handleClickPostModal = () => {
    setIsPostModalOpen(true);
  };
  
  const handleClosePostModal = () => {
    setIsPostModalOpen(false);
  };

  return (
```
import 구문 정리해주신것처럼 함수 안에서도
이런식의 구분이 들어가면 더 읽기 좋을거같아요

Reply:
훨씬 보기 편하군여,, 이런 부분도 신경써야겠어요 😃